### PR TITLE
chore(Changelog): Update change log for v21.03.1

### PR DESCRIPTION
**Pull request by ahsanbarkati**
---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7901)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,93 @@ All notable changes to this project will be documented in this file.
...
+## [21.03.1] - 2021-06-16
+[21.03.1]: https://github.com/dgraph-io/dgraph/compare/v21.03.0...v21.03.1
+
+### Fixed
+- GraphQL
+  - fix(GraphQL): fix @cascade with Pagination for @auth queries ([#7695][])
+  - Fix(GraphQL): Fix GraphQL encoding in case of empty list ([#7726][]) ([#7730][])
+  - Fix(GraphQL): Add filter in DQL query in case of reverse predicate ([#7728][]) ([#7733][])
+  - Fix(graphql): Fix error message of lambdaOnMutate directive  ([#7751][]) ([#7754][])
+  - opt(GraphQL): filter existence queries on GraphQL side instead of using @filter(type) ([#7757][]) ([#7760][])
```
Comment (NamanJain8) [inline CHANGELOG.md:18]: This is an optimization. Shouldn't it go to `Optimizations` section?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,93 @@ All notable changes to this project will be documented in this file.
...
+
+- Core Dgraph
+  - fix(vault): Hide ACL flags when not required ([#7701][])
+  - fix(Chunker): don't delete node with empty facet in mutation ([#7737][]) ([#7745][])
+  - fix(bulk): throw the error instead of crashing ([#7722][]) ([#7749][])
+  - fix(raftwal): take snapshot after restore ([#7719][]) ([#7750][])
+  - feat(cdc): Add support for SCRAM SASL mechanism ([#7765][]) ([#7767][])
```
Comment (NamanJain8) [inline CHANGELOG.md:25]: Move it to Features section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,93 @@ All notable changes to this project will be documented in this file.
...
+  - fix(txn): ensure that txn hash is set ([#7782][]) ([#7784][])
+  - bug fix to permit audit streaming to stdout writer([#7803][]) ([#7804][])
+  - fix(drop): attach galaxy namespace to drop attr done on 20.11 backup ([#7827][])
+  - fix: Prevent proposal from being dropped accidentally ([#7741][]) ([#7811][])
+  - fix(schema-update): Start opIndexing only when index creation is required. ([#7845][]) ([#7847][])
+  - fix(export): Fix facet export of reference type postings to JSON format ([#7744][]) ([#7756][])
+  - fix(lease): don't do rate limiting when not limit is not specified ([#7787][])
+  - fix(lease): prevent ID lease overflow ([#7802][])
+  - fix(auth): preserve the status code while returning error ([#7832][]) ([#7834][])
+  - chore(graphql): fixing query timeouts for graphql queries too ([#7796][])
```
Comment (NamanJain8) [inline CHANGELOG.md:36]: Remove chores.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,93 @@ All notable changes to this project will be documented in this file.
...
+  - Add asynchronous task API ([#7781][])
+  - fix(ee): GetKeys should return an error ([#7713][]) ([#7797][])
+  - fix(admin): remove exportedFiles field ([#7835][]) ([#7836][])
+  - feat(schema): do schema versioning and make backup non-blocking for i… ([#7856][]) ([#7873][])
+  - fix(restore): append galaxy namespace to type name ([#7881][])
+  - fix(DQL): revert changes related to cascade pagination with sort ([#7885][]) ([#7888][])
+  - make exports synchronous again ([#7877][])
```
Comment (NamanJain8) [inline CHANGELOG.md:43]: Move it to `Features` section? This change is not breaking, right?

**Review by NamanJain8 — APPROVED** [review]
Please address the comments before merge.

File: CHANGELOG.md (hunk 1 above)
Comment (ahsanbarkati) [reply CHANGELOG.md]: Moved to "Changed" section

**Pull request merged**